pgml-dashboard/Cargo.toml

```diff
@@ -17,20 +17,26 @@ base64 = "0.21"
 comrak = "0.17"
 chrono = "0.4"
 csv-async = "1"
+console-subscriber = "*"
 convert_case = "0.6"
 dotenv = "0.15"
 env_logger = "0.10"
+glob = "*"
 itertools = "0.10"
 parking_lot = "0.12"
 lazy_static = "1.4"
 log = "0.4"
-markdown = "1.0.0-alpha.13"
+markdown = "1.0.0-alpha.14"
 num-traits = "0.2"
 once_cell = "1.18"
+pgml = { version = "0.9.4", path = "../pgml-sdks/pgml/" }
+pgml-components = { path = "../packages/pgml-components" }
+pgvector = { version = "0.2.2", features = [ "sqlx", "postgres" ] }
 rand = "0.8"
 regex = "1.9"
+reqwest = { version = "0.11", features = ["json"] }
 rocket = { git = "https://github.com/SergioBenitez/Rocket", features = ["secrets", "json"] }
-sailfish = "0.8"
+sailfish = "0.8.0" # 0.8.1 has breaking changes
```

Comment:
What's the breaking change?

Reply:
Templates no longer compile, due to missing min/max args. It's not clear why, since those variables are not declared by our templates.

Reply:
Just need to clean the template cache. ```cargo clean```

Reply:
I think you may be right. I filed the issue with the authors: https://github.com/rust-sailfish/sailfish/issues/137

Reply:
Easy fix: https://github.com/rust-sailfish/sailfish/pull/138

Debugging is actually pretty fun!